tests/test_apel_plugin.py

```diff
@@ -293,6 +293,7 @@ def test_update_time_db_fail(self):
 
         with pytest.raises(Exception) as pytest_error:
             cur.execute("SELECT last_report_time FROM times")
+            cur.fetchall()
```

Comment:
Seems like python 3.11 is still fine with `cur.execute()` when one field contains a wrong type, and needs `cur.fetchall()` to fail as expected with `ValueError`.